catalog/java/io/material/catalog/navigationdrawer/CustomNavigationDrawerDemoActivity.java

```diff
@@ -86,24 +86,7 @@ public View onCreateDemoView(
         new SimpleDrawerListener() {
           @Override
           public void onDrawerOpened(@NonNull View drawerView) {
-            currentDrawerView = drawerView;
-            sideContainerBackHelper = new MaterialSideContainerBackHelper(drawerView);
-
-            if (VERSION.SDK_INT >= VERSION_CODES.UPSIDE_DOWN_CAKE) {
-              if (drawerOnBackAnimationCallback == null) {
-                drawerOnBackAnimationCallback = createOnBackAnimationCallback();
-              }
-              drawerLayout.post(
```

Comment:
@manabu-nakamura btw removing this post breaks the predictive back behavior in the normal flow - i.e. when a drawer is opened regularly without rotating the screen.

Reply:
Thank you. Does it break the predictive back behavior in the normal flow? There is no problem in Android Emulator 32.1.15 (Google APIs Intel x86_64 Atom System Image Revision 8).